src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -823,6 +823,13 @@ private bool ShouldTryToReconnect(Exception ex)
 
                     return true;
 
+                case DatabaseDisabledException:
+                case AllTopologyNodesDownException:
+                {
+                    AssertLastConnectionFailure();
+                    _redirectNode = null;
+                    return true;
```

Comment:
```suggestion
                        goto default;
```

Reply:
Shouldn't go to default because I want to prevent this:
https://github.com/ravendb/ravendb/blob/a70a5817605f97ef3062158549ccc1fe25726aa7/src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs#L871

Reply:
right, I guess we can drop `_redirectNode = null;` line then?

Reply:
Done